themes/src/main/resources-community/theme/base/login/messages/messages_es.properties

```diff
@@ -34,10 +37,16 @@ emailForgotTitle=\u00BFHas olvidado tu contrase\u00F1a?
 updatePasswordTitle=Modificaci\u00F3n de contrase\u00F1a
 codeSuccessTitle=C\u00F3digo de \u00E9xito
 codeErrorTitle=C\u00F3digo de error: {0}
+displayUnsupported=No se admite el tipo de visualizaci\u00F3n solicitado
+browserRequired=Se requiere un navegador para el inicio de sesi\u00F3n
+browserContinue=Se requiere un navegador para completar el inicio de sesi\u00F3n
+browserContinuePrompt=\u00BFAbrir el navegador y continuar  el inicio de sesi\u00F3n? [y/n]:
```

Comment:
Change "[y/n]" to "[s/n]"

Reply:
Agreed, as long as those are valid inputs :laughing: